SUPPORTED_CLIENTS.md

```diff
@@ -28,14 +28,14 @@ These are the clients we actively test against to check are compatible with go-m
 import pymysql.cursors
 
 connection = pymysql.connect(host='127.0.0.1',
-                             user='user',
-                             password='pass',
-                             db='db',
+                             user='root',
+                             password='',
+                             db='gitbase',
                              cursorclass=pymysql.cursors.DictCursor)
 
 try:
     with connection.cursor() as cursor:
-        sql = "SELECT foo FROM bar"
+        sql = "SELECT * FROM commit_files LIMIT 1"
```

Comment:
These are go-mysql-server examples, not gitbase examples. I don't think we should mix the two of them